### PR TITLE
Implement test case "Verify that a user is able to login/logout in di…

### PR DESCRIPTION
**Pull request by viorelaioia-zz**
…scourse"

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,26 @@
...
+
+How to set up and run Discourse tests locally
+
+This repository contains Selenium tests used to test:
+* staging: https://discourse.staging.paas.mozilla.community/
+* production: https://discourse.mozilla-community.org/
+
+###You will need to install the following:
+* Git
+If you have cloned this project already then you can skip this! GitHub has excellent guides for Windows, Mac OS X, andLinux.
```
Comment (johngian) [inline README.md]: Please fix typo ``andLinux``.

File: README.md [hunk 2]
```diff
@@ -1,2 +1,26 @@
...
+
+* Python
+Before you will be able to run these tests you will need to have Python 2.6.8+ installed.
```
Comment (johngian) [inline README.md:15]: Markdown doesn't treat this as a new line, please make sure that the rendered doc looks OK.

File: README.md [hunk 3]
```diff
@@ -1,2 +1,26 @@
...
+
+###Running tests locally
+* Credentials
+Some of the tests in discourse-tests require accounts for https://discourse.staging.paas.mozilla.community.  You'll need to create two sets of credentials with varying privilege levels.
```
Comment (johngian) [inline README.md:18]: Same here

File: README.md [hunk 4]
```diff
@@ -1,2 +1,26 @@
...
+1. Create Two username and password combinations on https://mozillians.org
```
Comment (johngian) [inline README.md]: ``Two`` --> ``two``

File: README.md [hunk 5]
```diff
@@ -1,2 +1,26 @@
...
+2. Make sure one of the accounts is vouched 3 times
+3. Create the same Two accounts used in step #1 in https://discourse.staging.paas.mozilla.community
```
Comment (johngian) [inline README.md:22]: same here

File: README.md [hunk 6]
```diff
@@ -1,2 +1,26 @@
 # discourse-tests
 Automated tests for discourse.mozilla-community.org
+
+How to set up and run Discourse tests locally
+
+This repository contains Selenium tests used to test:
+* staging: https://discourse.staging.paas.mozilla.community/
+* production: https://discourse.mozilla-community.org/
+
+###You will need to install the following:
```
Comment (johngian) [inline README.md]: nit: Why don't you use ``##`` instead of ``###`` to keep the semantics correct

File: README.md (hunk 2 above)
Comment (johngian) [reply README.md:15]: This has other occurrences too

File: pages/base.py [hunk 7]
```diff
@@ -0,0 +1,46 @@
+from selenium.webdriver.common.by import By
+from selenium.webdriver.support.ui import WebDriverWait
+
+from pages.page import Page
+from pages.auth0 import Auth0
```
Comment (johngian) [inline pages/base.py:4]: Please move this one line up. Its better to keep imports in alphabetic order.

File: tests/restmail.py [hunk 8]
```diff
@@ -0,0 +1,29 @@
+import json
+import time
+
+import requests
+
+
+def get_mail(username, message_count=1, timeout=60):
+    username = username.partition('@restmail.net')[0]
```
Comment (johngian) [inline tests/restmail.py:12]: Please add a docstring to explain what this method does.

**Review by johngian — CHANGES_REQUESTED** [review]

**Comment by johngian** [comment]
Please squash commits to a single one.

File: README.md (hunk 6 above)
Comment (viorelaioia-zz) [reply README.md]: updated

File: README.md (hunk 1 above)
Comment (viorelaioia-zz) [reply README.md]: fixed

File: README.md (hunk 4 above)
Comment (viorelaioia-zz) [reply README.md]: updated

File: pages/base.py (hunk 7 above)
Comment (viorelaioia-zz) [reply pages/base.py:4]: updated

File: tests/restmail.py (hunk 8 above)
Comment (viorelaioia-zz) [reply tests/restmail.py:12]: updated

**Review by johngian — APPROVED** [review]

**Pull request merged**